.github/workflows/package.yml

```
@@ -12,8 +12,14 @@ permissions: write-all
 
 jobs:
   build:
-
-    runs-on: windows-2019
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
```

```suggestion
        os: [windows-2019, ubuntu-latest, macos-latest]
```

这个地方我还是倾向于用老版本系统，因为 GUI 版的兼容目标是 Windows 10 1809